conf/application.conf

```diff
@@ -20,7 +20,7 @@ include "backend.conf"
 appName = register-ncf-stub
 
 # Default http client
-play.modules.enabled += "uk.gov.hmrc.play.bootstrap.HttpClientModule"
+play.modules.enabled += "uk.gov.hmrc.play.bootstrap.HttpClientV2Module"
```

Comment:
minor: but we can remove this completely as we dont have any connectors in this service

Reply:
Thanks Suresh